python/hail/docs/tutorials/hail-overview.ipynb

```diff
@@ -1118,9 +1118,9 @@
    "source": [
     "## Rare variant analysis\n",
     "\n",
-    "Hail doesn't yet have rare variant kernel-based methods, but we have [linear](https://hail.is/hail/hail.VariantDataset.html#hail.VariantDataset.linreg_burden) and [logistic](https://hail.is/hail/hail.VariantDataset.html#hail.VariantDataset.logreg_burden) burden tests. \n",
+    "Hail doesn't yet have rare variant kernel-based methods.\n",
```

Comment:
How about:
"Here we'll demonstrate how one can use the expression language to group and count by any arbitrary properties in variant or sample annotations. Hail also implements the sequence kernel association test (SKAT)."